internal/version/version.go

```diff
@@ -13,7 +13,7 @@ import (
 // Tag specifies the current release tag. It needs to be manually
 // updated. A test checks that the value of Tag never points to a
 // git tag that is older than HEAD.
-const Tag = "v1.62.0-dev"
+const Tag = "v1.62.0-rc.1"
```

Comment:
I don't suppose this is expected to be there?

Reply:
Correct - looking at the PR, an extra commit from someone else made it onto this branch and I don't understand how.